src/SwedbankPay.Sdk.Infrastructure/PaymentInstruments/Vipps/VippsPaymentsResource.cs

```diff
@@ -0,0 +1,28 @@
+﻿using SwedbankPay.Sdk.Payments.VippsPayments;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments
+{
+    public class VippsPaymentsResource : ResourceBase, IVippsResource
+    {
+        public VippsPaymentsResource(HttpClient httpClient) : base(httpClient)
+        {
+        }
+
+        public Task<IVippsPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None)
+        {
+            if (id == null)
+                throw new ArgumentNullException(nameof(id));
+
+            return VippsPayment.Get(id, this.HttpClient, GetExpandQueryString(paymentExpand));
+        }
+
+        public Task<IVippsPayment> Create(VippsPaymentRequest paymentRequest,
+                                                            PaymentExpand paymentExpand = PaymentExpand.None)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)